docs/zk-passport/guide-proof-of-citizenship.mdx

```diff
@@ -40,65 +42,68 @@ Links for installation:
 - <OutLink href="https://apps.apple.com/us/app/rarime/id6503300598">RariMe for iOS | AppStore</OutLink>
 - <OutLink href="https://play.google.com/store/apps/details?id=com.rarilabs.rarime">RariMe for Android | Google Play</OutLink>
 
-## Step #2: Render a QR code with a verification request in your app
+## Step #2: Install `@rarimo/zk-passport-react` package
 
-To request data for the QR code, you need to call this endpoint on the `verificator-svc`:
-<OutLink href="https://api.app.rarime.com/integrations/verificator-svc/private/verification-link">https://api.app.rarime.com/integrations/verificator-svc/private/verification-link</OutLink>
+To install the package, run the following command in your project directory:
 
-The following JS snippet gets data for a QR code for a passport verification with a uniqueness check on:
+```bash npm2yarn
+npm install @rarimo/zk-passport-react
+```
 
-```jsx
-const verificatorUrl = 'https://api.app.rarime.com/' // your verificator-svc instance URL
-const body = {
-  data: {
-	  type: "user",
-    // Some public and unique user ID
-    // Will be used to fetch status later
-    id: "some-user-id",
-    attributes: {
-      // Denotes the application and activity that is requesting the verification
-      event_id: "citizenship_verification",
-      // Let a single passport pass verification just once
-      uniqueness: true
-    }
-  }
-}
+If you don't use React, you can use the `@rarimo/zk-passport` package instead. It provides a low-level API for for requesting and passing for verification of ZK Passport proofs.
 
-const response = await fetch(
-
-  `${verificatorUrl}/integrations/verificator-svc/private/verification-link`,
-  {
-    headers: {
-      'Accept': 'application/vnd.api+json',
-      'Content-Type': 'application/vnd.api+json',
-    },
-    method: "POST",
-    body: JSON.stringify(body),
-  }
-)
-const { data } = await response.json()
-// Proof Params URL for RariMe app
-const proofParamsUrl = data.attributes.get_proof_params
-const encodedUrl = encodeURIComponent(proofParamsUrl)
-// Render this data in a QR code:
-const qrCodeContent = `rarime://external?type=proof-request&proof_params_url=${encodedUrl}`)
+```bash npm2yarn
+npm install @rarimo/zk-passport
 ```
 
-Some notes:
+## Step #3: Import the library and render the QR code
+
+The following JS snippet uses the library to render a QR code with a proof of citizenship request and handles the results:
 
-- `event_id` is an arbitrary string that denotes the application scope of the proof, should be something like `APP_ID:ACTION_ID`
-- `id` is a public unique ID associated with a user within your system. It will be used in Step #3 to check the user status
+```jsx
+import ZkPassportQrCode, { ProofRequestStatuses } from '@rarimo/zk-passport-react'
```

Comment:
ProofRequestStatuses is not used

Reply:
Removed. Pls fix it in the SDK readme too